charts/newrelic-infrastructure/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 description: "DEPRECATED: (In maintenance mode) Helm chart for deploying New Relic Infrastructure Agent v2"
 name: newrelic-infrastructure
-version: 2.10.5
-appVersion: 2.13.2
+version: 2.10.6
+appVersion: 2.13.3
```

Comment:
we are not bundling the last one, is it intended?

Reply:
no this is an error i will correct , i just bumped one patch assuming it was correct